docs/infrastructure/notifications.md

```diff
@@ -0,0 +1,37 @@
+| Source                          | Environment        | Trigger                                   | Slack                  | Schedule |
```

Comment:
Maybe change the `Slack` heading to be `destination` and specify which alerts are being routed through OpsGenie